tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -81,7 +79,7 @@ public partial class LandingPageViewModel : ObservableObject, IDisposable
     [ObservableProperty]
     private bool _shouldShowCreationHeader;
 
-    private const int SortUnselected = -1;
+    private const int DefaultSortIndex = 2;

```

Comment:
The numeric value used here has no meaning in this context. You'd have to follow to the implementation of SortHandler in another class to understand what it means. Consider defining an enum (probably in PerProviderViewModel) with possible sort values.

Reply:
Added enum.